### DISCUSSION TITLE
能否增加多个账号的云盘

### DISCUSSION THREAD

**Discussion by ElvenP** [messense/aliyundrive-webdav#100] [Ideas]
如果能增加多个账号云盘这样就更好了。

**Comment by messense** [comment]
可以直接[命令行](https://github.com/messense/aliyundrive-webdav#%E5%91%BD%E4%BB%A4%E8%A1%8C%E7%94%A8%E6%B3%95)起多个进程不同端口，暂不考虑。

**Comment by Glenffi** [comment]
openwrt版本同样希望能够在luci界面上支持多账号

**Reply by swswlx** [reply]
赞同

**Reply by wangshengxin1998** [reply]
希望能做到多账号端口复用,使用不同的用户名密码组合来访问不同的账号

**Reply by clbin001** [reply]
赞同，期待！

**Reply by acoooder** [reply]
同求，期待

**Reply by nuoooo** [reply]
同求

**Reply by liyiranlab** [reply]
现在用了多个路由器分别挂载一家人的账号，如果有多账号功能，经常升级的路由器就不用挂载，专门用不常升级的路由器挂载会减少一些重复性的麻烦。

**Comment by shawphy** [comment]
希望可以多个账号分别挂载在不同的目录下。

**Comment by acoooder** [comment]
> 如果能增加多个账号云盘这样就更好了。
ssh连上，用命令行起一个就行了，端口和refresh-token、workdir区分开就行：
nohup /usr/bin/aliyundrive-webdav --auto-index --host **** --port **** --refresh-token **** --root / --read-buffer-size 10485760 --upload-buffer-size 16777216 --cache-size 1000 --cache-ttl 60 --workdir /var/run/**aliyundrive-webdav**   &

**Comment by trueai-org** [comment]
@acoooder @shawphy @ElvenP  这个提供加密版，多账号支持。
https://github.com/trueai-org/mdrive

多平台、模块化、可挂载、安全、加密的云盘自动同步、备份工具，支持本地存储、阿里云盘（多账号）等，支持单向、镜像、双向等同步备份，完全免费开源。

提供 Docker、Windows、Linux、Web 等多平台版本。

**支持 AES256-GCM、ChaCha20-Poly1305 加密，支持 Zstd、LZ4、Snappy 压缩，支持 SHA256、BLAKE3 等哈希算法，任何第三方、服务商都无法查看数据，保护您的数据安全和隐私。**
